src/OpenSage.Game.Tests/Data/Sav/SaveFileTests.cs

```diff
@@ -43,6 +43,20 @@ public void CanLoadSaveFiles(string relativePath)
             game.EndGame();
         }
 
+        [Fact]
+        public void CanLoadTestWeapons()
```

Comment:
Is it necessary to add this test? I think it should be covered by `CanLoadSaveFiles` above.

Reply:
Right, I added it because the other ones were disabled, but I guess it can be removed.

Reply:
Ah yes, locally I have this change:

``` csharp
//[Theory(Skip = "Doesn't work yet")]
[Theory]
[MemberData(nameof(GetSaveFiles))]
public void CanLoadSaveFiles(string relativePath)
```

Reply:
I think we can enable those tests by default soon - for me locally, there are only 3 failing ones, which happen to be failing on a road-related issue, not a .sav parsing issue ;-)